ui.frontend/src/view/FormFieldBase.js

```diff
@@ -90,7 +96,9 @@ export default class FormFieldBase extends FormField {
      * @private
      */
     _updateVisible(visible) {
-        this.toggle(visible, this.dataAttribute('hidden'), true)
+        this.toggle(visible, this.dataAttribute('hidden'), true);
+        this.toggle(visible, Constants.ARIA_HIDDEN, true);
+        this.toggle(visible, Constants.VISIBLE, false);
```

Comment:
Please remove line 99 in that case. Also can you check if boolean attribute data-cmp-visible=false is supported in HTML ? For boolean attributes, the attribute should be present

Reply:
And update the css classes as well

Reply:
Will update css. data-cmp-visible=false can exist, but i believe toggle is removing the attribute, which works as well. I am only keeping bot for active that is, data-cmp-active=true and data-cmp-active=false